### DISCUSSION TITLE
What is the difference between "Show lowest battery level" and "Show average battery level" in "Dynamic tray icon"?

### DISCUSSION THREAD

**Discussion by Weltolk** [timschneeb/GalaxyBudsClient#509] [Q&A] [answered]
![image](https://github.com/timschneeb/GalaxyBudsClient/assets/40228052/33988f5d-86be-4221-824f-3698ebe68f0d)

**Comment by timschneeb** [comment, marked as answer]
For example, if the left earbud has a 10% charge and the right earbud has a 20% charge:
* show lowest battery level will display 10% because the left earbud has the lowest charge of the two
* show average battery level will display 15% because that's the average charge of both earbuds
